client/app/components/packages/pas-form/edit.hbs

```diff
@@ -727,6 +734,7 @@
                   <button
                     type="button"
                     class="button large secondary no-margin"
+                    disabled={{this.submit.isRunning}}
```

Comment:
Is this so that the user doesn't click out of the modal after they've said they want to submit _for sure_?

Reply:
There's nothing preventing the user from closing the modal by clicking outside of it. The div that's behind the modal (which grays out the page) also has the `toggleModal` action. Perhaps `toggleModal` needs to consider `isRunning`?

Reply:
I think for a few reasons: 
   - So they can't double-click "Submit" 
   - Consistency in UI experience/design. The UI will consistently teach the user that disabled buttons +  spinner indicates there is "network/databse activity" and there shouldn't be any more interaction.

Reply:
@andycochran  oh good point. I can add that to toggleModal